README.md

```diff
@@ -510,9 +516,20 @@ To compress PNGs and SVGs in the current directory and its subdirectories, two c
 
 # Releasing
 
-1. Create a pull request. Get it reviewed and approved.
-1. Run `npm version patch`, `npm version minor`, or `npm version major` as appropriate.
-1. Merge the pull request.
+## Do I need to update `release-notes.md`?
+
+As we strive to have clear, readable commit messages for every change, [release-notes.md](./release-notes.md) should not be updated for every single change. Instead, it should be used as a forum for deep information about significant changes, breaking changes, changes to interfaces, changes in behavior, new feature details, etc.
+
+## Performing the release
+
+1. Create a pull request with prep for the new release. This should contain changes to release notes if required and anything else that needs to be done:
```

Comment:
Sometimes the only content of the PR would be the result of the npm version command...  which gets a little weird

Also: should we clarify that the user should ensure master is synced or their working branch is rebased to master?